frontend/dispatcher.lua

```diff
@@ -1018,6 +1019,31 @@ function Dispatcher:addSubMenu(caller, menu, location, settings)
             end
         end,
     })
+    table.insert(menu, {
+        text = _("Keep QuickMenu open on apply"),
```

Comment:
("on apply" feels a bit technical/internal jargon to me - good in the code, dunno if it sounds alright to users. may be just "Keep QuickMenu open" ?)

Reply:
Don't we have a similar phrasing for that somewhere already? But yes, definitely drop the "on apply".